apps/mobile/src/features/send/send-form/hooks/use-send-form-btc.tsx

```diff
@@ -12,56 +13,76 @@ import {
   CoinSelectionRecipient,
   CoinSelectionUtxo,
   getBitcoinFees,
+  isValidBitcoinTransaction,
   payerToBip32Derivation,
 } from '@leather.io/bitcoin';
-import { AverageBitcoinFeeRates } from '@leather.io/models';
+import {
+  AverageBitcoinFeeRates,
+  Money,
+  bitcoinNetworkToNetworkMode,
+  createBitcoinAddress,
+} from '@leather.io/models';
 import { createMoneyFromDecimal } from '@leather.io/utils';
 
 import {
   CreateCurrentSendRoute,
   createCoinSelectionUtxos,
-  formatBitcoinError,
   useSendSheetNavigation,
   useSendSheetRoute,
 } from '../../send-form.utils';
 import { SendFormBtcContext } from '../providers/send-form-btc-provider';
 import { SendFormBtcSchema } from '../schemas/send-form-btc.schema';
+import { formatBitcoinError } from '../validation/btc.validation';
 
 type CurrentRoute = CreateCurrentSendRoute<'send-form-btc'>;
 
-function parseSendFormValues(values: SendFormBtcSchema) {
+interface ParsedSendFormValues {
+  amount: Money;
+  recipients: CoinSelectionRecipient[];
+}
+
+function parseSendFormValues({
+  amount: inputAmount,
+  recipient,
+}: SendFormBtcSchema): ParsedSendFormValues {
+  const amount = createMoneyFromDecimal(new BigNumber(inputAmount), 'BTC');
   return {
-    amount: createMoneyFromDecimal(new BigNumber(values.amount), 'BTC'),
+    amount,
     recipients: [
       {
-        address: values.recipient,
-        amount: createMoneyFromDecimal(new BigNumber(values.amount), 'BTC'),
+        address: createBitcoinAddress(recipient),
+        amount,
       },
     ],
   };
 }
-
 interface GetTxFeesArgs {
   feeRates: AverageBitcoinFeeRates;
   recipients: CoinSelectionRecipient[];
   utxos: CoinSelectionUtxo[];
 }
-
 export function useSendFormBtc() {
-  const route = useSendSheetRoute<CurrentRoute>();
+  const {
+    params: { account, address, publicKey },
+  } = useSendSheetRoute<CurrentRoute>();
   const navigation = useSendSheetNavigation<CurrentRoute>();
-  const { account } = route.params;
   const { displayToast } = useToastContext();
+  const {
+    networkPreference: {
+      chain: {
+        bitcoin: { bitcoinNetwork },
+      },
+    },
+  } = useSettings();
+  const payer = createBitcoinAddress(address);
```

Comment:
`address` is a route `param` so it would likely make more sense to set the `address` as `BitcoinAddress` there too